metricbeat/module/couchbase/cluster/_meta/fields.yml

```diff
@@ -1,62 +1,60 @@
 - name: cluster
   type: group
   description: >
-    experimental[]
```

Comment:
Reviewers: In a couple of places, I've removed the `experimental[ ]` tags because they resulted in multiple instances of the note within exported-fields-couchbase.html. I think that one note at the top of the page is sufficient since all the metricsets are experimental right now.